content/scout/sbom.md

```diff
@@ -4,9 +4,8 @@ description: Use Docker Scout to extract the SBOM for your project.
 keywords: scout, supply chain, sbom, software bill of material, spdx
 ---
 
-As part of [image analysis](./image-analysis.md), Docker Scout generates a
-Software Bill of Material (SBOM) for your project. The SBOM uses the Software
-Package Data Exchange (SPDX) format.
+As part of [image analysis](./image-analysis.md),
+Docker Scout generates a Software Bill of Material (SBOM) of your image.
```

Comment:
I think we could encourage users to start building images with attestations here, which is our recommendation . Might just be a simple "... or ingests an SBOM if one is availble for the image (recommended)"